File: testdata/clean/sample.java
public class Sample {
    // Regular class comment
    public static void main(String[] args) {
        System.out.println("Hello, World!");
        
        /* Standard multi-line comment
           explaining the logic */
        connectToDatabase();
    }
    
    // Method documentation
    private static void connectToDatabase() {
        // Implementation details
        String password = getPasswordFromEnv();
    }
    
    private static String getPasswordFromEnv() {
        return System.getenv("DB_PASSWORD");
    }
}

File: testdata/clean/sample.go
package testdata

import "fmt"

// Regular comment explaining the function
func TestFunction() {
	fmt.Println("Hello, World!")
	// This is a normal comment
	fmt.Println("Debug information")
	x := 42
}

// Another regular comment
func AnotherFunction() {
	// TODO: This is fine - not a bark comment
	x := 42
	fmt.Println(x)
}
